t/not_implemented_hook.t

```diff
@@ -0,0 +1,48 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+
+use Mojolicious::Lite;
+
+plugin OpenAPI => {
+  url => 'data://main/emulation.json',
+  not_implemented => sub {
+    my ( $c,$spec ) = @_;
```

Comment:
Please tidy all the files. There's a default .perltidyrc you can use in the root of this project.